README.md

```diff
@@ -19,215 +19,160 @@ REQUIRED SECTIONS:
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 Morse-Code<a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
+> Morse-Code is our first project in ruby programming language and it convert morse message to human readable message.
 
-**[your_project__name]** is a...
+**Morse-Code** is the ruby methods project that convert morse messages.
 
 ## 🛠 Built With <a name="built-with"></a>
 
-### Tech Stack <a name="tech-stack"></a>
+- Ruby
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
+### Tech Stack <a name="tech-stack"></a>
 
 <details>
   <summary>Client</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://N/A.org/">N/A</a></li>
   </ul>
 </details>
 
 <details>
   <summary>Server</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li>Ruby</li>
   </ul>
 </details>
 
 <details>
 <summary>Database</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li>N/A</li>
   </ul>
 </details>
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Decode Morse Messages**
+- **Using hashes to perform the operations**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link Comming Soon!]
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
 In order to run this project you need:
 
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
+- Laptop/Desktop.
+- Internet.
+- Ruby.
+- Vs Code.
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-<!--
-Example commands:
-
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
-```
---->
+- mkdir my-folder
+- cd my-folder
+- git clone [(https://github.com/Trust-007/Morse-Code.git)]
 
 ### Install
 
 Install this project with:
 
-<!--
-Example command:
+**you can run any one of these commands to install npm on your machine:**
 
-```sh
-  cd my-project
-  gem install
-```
---->
+- npm i
+- npm install
+- sudo apt install npm
 
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you update the content of the `Install` section to make your documentation more professional. You add here commands related to a `ruby` application(e.g.: `bundle install`,...)
I suggest you apply this approach to other sections.